### DISCUSSION TITLE
Breadcrumbs on node server with SSR

### DISCUSSION THREAD

**Discussion by lajennylove** [felix-berlin/astro-breadcrumbs#204] [Q&A]
Hello there guys… Do you have any idea why breadcrumbs created with your library seems to work well using "npm run dev" or using netlify or vercel. 
![image](https://github.com/felix-berlin/astro-breadcrumbs/assets/5205775/9996f1b2-254e-41bc-b4a6-8b8d243681e8)

But it doesn’t, when you use the SSR with node server implementation?

`node ./dist/server/entry.mjs`
 
![image](https://github.com/felix-berlin/astro-breadcrumbs/assets/5205775/1384eadb-6cd7-4efb-a548-68f548a203f1)

I'm using the latest Astro version and this is my astro.config.mjs

```JavaScript
import { defineConfig } from 'astro/config';
import tailwind from '@astrojs/tailwind';
import node from "@astrojs/node";

export default defineConfig({
  integrations: [tailwind()],
  base: '/the-beach',
  trailingSlash: "ignore",
  output: 'server',
  adapter: node({
    mode: 'standalone',
  }),
});
```

**Comment by felix-berlin** [comment]
Hi @lajennylove I'm not sure why this happens 😬

To be honest this component was never tested with SSR and all those adapters.

I do my best to fix this issue.
